### ISSUE TITLE
Extend Exif data recognition 

### ISSUE THREAD

**Issue by mwjames** [SemanticMediaWiki/SemanticExtraSpecialProperties#16]
Using [0] as basis together with the required merge of #10, `definitions.json` can be extended with the data seen in [1] that would allow for additional exif data to appear in the `exif` subobject without further development effort. A guideline on how to maintain the `definitions.json` can be found [here](https://github.com/SemanticMediaWiki/SemanticExtraSpecialProperties/pull/10#issuecomment-35386089).

[0] [git.wikimedia.org - SemanticExtraSpecialProperties.hooks](https://git.wikimedia.org/raw/mediawiki%2Fextensions%2FSemanticExtraSpecialProperties/9a226e294ce529cfe3aa300fcf1db0f237a2779e/SemanticExtraSpecialProperties.hooks.php)

[1] 

```
    static private $mPropertyTypesToExifTags = array(
        '_num' => array(
            'OriginalImageHeight',
            'OriginalImageWidth',
            'PixelXDimension',
            'PixelYDimension',
            'ImageWidth',
            'ImageLength',
            'Rating',
        ),
        '_dat' => array(
            'DateTime',
            'DateTimeOriginal',
            'DateTimeDigitized',
            'DateTimeReleased',
            'DateTimeExpires',
            'GPSDateStamp',
            'dc-date',
            'DateTimeMetadata',
        ),
        '_txt' => array(
            'Compression',
            'PhotometricInterpretation',
            'Orientation',
            'PlanarConfiguration',
            'YCbCrPositioning',
            'XResolution',
            'YResolution',
            'ExifVersion',
            'FlashpixVersion',
            'ColorSpace',
            'ComponentsConfiguration',
            'ExposureProgram',
            'SubjectDistance',
            'MeteringMode',
            'LightSource',
            'Flash',
            'FocalPlaneResolutionUnit',
            'SensingMethod',
            'FileSource',
            'SceneType',
            'CustomRendered',
            'ExposureMode',
            'WhiteBalance',
            'SceneCaptureType',
            'GainControl',
            'Contrast',
            'Saturation',
            'Sharpness',
            'SubjectDistanceRange',
            'Make',
            'Model',
            'Software',
            'ExposureTime',
            'ISOSpeedRatings',
            'FNumber',
            'FocalLength',
            'FocalLengthIn35mmFilm',
            'MaxApertureValue',
            'iimCategory',
            'SubjectNewsCode',
            'Urgency',
            'ImageDescription',
            'Artist',
            'Copyright',
            'RelatedSoundFile',
            'ImageUniqueID',
            'SpectralSensitivity',
            'GPSSatellites',
            'GPSVersionID',
            'GPSMapDatum',
            'Keywords',
            'WorldRegionDest',
            'CountryDest',
            'CountryCodeDest',
            'ProvinceOrStateDest',
            'CityDest',
            'SublocationDest',
            'WorldRegionCreated',
            'CountryCreated',
            'CountryCodeCreated',
            'ProvinceOrStateCreated',
            'CityCreated',
            'SublocationCreated',
            'ObjectName',
            'SpecialInstructions',
            'Headline',
            'Credit',
            'Source',
            'EditStatus',
            'FixtureIdentifier',
            'LocationDest',
            'LocationDestCode',
            'Writer',
            'JPEGFileComment',
            'iimSupplementalCategory',
            'OriginalTransmissionRef',
            'Identifier',
            'dc-contributor',
            'dc-coverage',
            'dc-publisher',
            'dc-relation',
            'dc-rights',
            'dc-source',
            'dc-type',
            'Lens',
            'SerialNumber',
            'CameraOwnerName',
            'Label',
            'Nickname',
            'RightsCertificate',
            'CopyrightOwner',
            'UsageTerms',
            'WebStatement',
            'OriginalDocumentID',
            'LicenseUrl',
            'MorePermissionsUrl',
            'AttributionUrl',
            'PreferredAttributionName',
            'PNGFileComment',
            'Disclaimer',
            'ContentWarning',
            'GIFFileComment',
            'SceneCode',
            'IntellectualGenre',
            'Event',
            'OrginisationInImage',
            'PersonInImage',
            'ObjectCycle',
            'Copyrighted',
            'LanguageCode',
        ),
    );
```

**Comment by mwjames** [comment]
Addressed with #75.

**Comment by kghbln** [comment]
> Addressed with #75.

Amazing. I let this rot. :( and you did the labour. [You are my hero](http://mybrainllc.com/wp-content/uploads/2015/04/hero.jpg)
